src/pages/Home/index.js

```diff
@@ -185,26 +185,32 @@ const Announcement = () => (
           className={f('small', 'icon', 'icon-common', 'icon-announcement')}
         />{' '}
         <p>
-          <strong>Powering down the Pfam website</strong>
+          <strong>We are the new home for Pfam data</strong>
           <br />
-          On October 5th, we will start redirecting the traffic from Pfam (
-          <Link href="//pfam.xfam.org" className={f('ext')} target="_blank">
+          Since October 5th, we are the official web for Pfam. The Pfam website
+          (
+          <span href="//pfam.xfam.org" className={f('link', 'disabled')}>
             pfam.xfam.org
-          </Link>
-          ) to InterPro (
+          </span>
+          ) is now retired.
+          <br />A legacy version of the Pfam website will be available at{' '}
           <Link
-            to={{
-              description: {},
-            }}
+            href="//pfam-legacy.xfam.org/"
+            className={f('ext')}
+            target="_blank"
+          >
+            https://pfam-legacy.xfam.org/
+          </Link>{' '}
+          until January 2023, when it will be decommissioned. Please notice{' '}
+          <Link
+            href="//pfam-legacy.xfam.org/"
+            className={f('ext')}
+            target="_blank"
           >
-            www.ebi.ac.uk/interpro
-          </Link>
-          ). The Pfam website will be available at{' '}
-          <span href="//pfam-legacy.xfam.org" className={f('link', 'disabled')}>
             pfam-legacy.xfam.org
-          </span>{' '}
-          until January 2023, when it will be decommissioned. You can read more
-          about the sunset period in our{' '}
+          </Link>{' '}
+          will not receive any updates and will be kept under very low
+          maintenance. You can read more about the sunset period in our{' '}
           <Link
             href="//xfam.wordpress.com/2022/08/04/pfam-website-decommission/"
             className={f('ext')}
```

Comment:
Suggestion for text:

**InterPro is the new home of Pfam**
The Pfam website (https://pfam.xfam.org) was shut down on October 5th, but InterPro offers the same functionality and data. A legacy version of Pfam will remain available at https://pfam-legacy.xfam.org until January 2023, but will not receive any updates. You can read more about our decision to shut down the Pfam website in [our blog post ](xfam.wordpress.com/2022/08/04/pfam-website-decommission).